main.py

```diff
@@ -15,7 +15,6 @@
 import sys
 import os
 import cmd
-from models.room import Room
 from docopt import docopt, DocoptExit
 from models.dojo import Dojo
```

Comment:
NIT: You should have a space between your inbuilt libraries, 3rd party libraries and project specific libraries import. 
Also, you should import in alphabetical order 

(that if you give a rat's ass about [PEP8 Standards](https://www.python.org/dev/peps/pep-0008/))

But seriously, this is recommended